core/lib/TabNavigationsObserver.ts

```diff
@@ -221,7 +221,7 @@ export default class TabNavigationsObserver {
     if (this.waitingForPromise) this.cancelWaiting('New location trigger created');
 
     this.waitingForStatus = status;
-    this.waitingForPromise = createPromise<void>(timeoutMs ?? 30e3);
+    this.waitingForPromise = createPromise<void>(timeoutMs ?? 60e3);
```

Comment:
double the default timeout?

Reply:
DOMContentLoaded has been recorded taking 34 and 32 seconds when I was hitting the same url over and over.... so I moved to 60. I don't like having a timeout here with a specific value, but it kind of needs to be something.